### ISSUE TITLE
fixtures:execute error

### ISSUE THREAD

**Issue by arhimede** [dotkernel/api#297]
latest api code from 5.0 branch
following installation instrutions

at the step:
`php bin/doctrine fixtures:execute`

I got 

> PHP Fatal error:  Uncaught Error: Call to undefined method Api\Admin\Entity\AdminRole::updateTimestamps() in /var/www/api.dotkernel.localhost/html/vendor/doctrine/orm/src/Event/ListenersInvoker.php:80
Stack trace:
#0 /var/www/api.dotkernel.localhost/html/vendor/doctrine/orm/src/UnitOfWork.php(909): Doctrine\ORM\Event\ListenersInvoker->invoke()
#1 /var/www/api.dotkernel.localhost/html/vendor/doctrine/orm/src/UnitOfWork.php(1802): Doctrine\ORM\UnitOfWork->persistNew()
#2 /var/www/api.dotkernel.localhost/html/vendor/doctrine/orm/src/UnitOfWork.php(1760): Doctrine\ORM\UnitOfWork->doPersist()
#3 /var/www/api.dotkernel.localhost/html/vendor/doctrine/orm/src/EntityManager.php(434): Doctrine\ORM\UnitOfWork->persist()
#4 /var/www/api.dotkernel.localhost/html/data/doctrine/fixtures/AdminRoleLoader.php(20): Doctrine\ORM\EntityManager->persist()
#5 /var/www/api.dotkernel.localhost/html/vendor/doctrine/data-fixtures/src/Executor/AbstractExecutor.php(122): Api\Fixtures\AdminRoleLoader->load()
#6 /var/www/api.dotkernel.localhost/html/vendor/doctrine/data-fixtures/src/Executor/ORMExecutor.php(26): Doctrine\Common\DataFixtures\Executor\AbstractExecutor->load()
#7 /var/www/api.dotkernel.localhost/html/vendor/doctrine/orm/src/EntityManager.php(182): Doctrine\Common\DataFixtures\Executor\ORMExecutor::Doctrine\Common\DataFixtures\Executor\{closure}()
#8 /var/www/api.dotkernel.localhost/html/vendor/doctrine/data-fixtures/src/Executor/ORMExecutor.php(20): Doctrine\ORM\EntityManager->wrapInTransaction()
#9 /var/www/api.dotkernel.localhost/html/vendor/dotkernel/dot-data-fixtures/src/Command/ExecuteFixturesCommand.php(74): Doctrine\Common\DataFixtures\Executor\ORMExecutor->execute()
#10 /var/www/api.dotkernel.localhost/html/vendor/symfony/console/Command/Command.php(279): Dot\DataFixtures\Command\ExecuteFixturesCommand->execute()
#11 /var/www/api.dotkernel.localhost/html/vendor/symfony/console/Application.php(1029): Symfony\Component\Console\Command\Command->run()
#12 /var/www/api.dotkernel.localhost/html/vendor/symfony/console/Application.php(316): Symfony\Component\Console\Application->doRunCommand()
#13 /var/www/api.dotkernel.localhost/html/vendor/symfony/console/Application.php(167): Symfony\Component\Console\Application->doRun()
#14 /var/www/api.dotkernel.localhost/html/vendor/doctrine/orm/src/Tools/Console/ConsoleRunner.php(30): Symfony\Component\Console\Application->run()
#15 /var/www/api.dotkernel.localhost/html/bin/doctrine(23): Doctrine\ORM\Tools\Console\ConsoleRunner::run()
#16 {main}
  thrown in /var/www/api.dotkernel.localhost/html/vendor/doctrine/orm/src/Event/ListenersInvoker.php on line 80

**Comment by alexmerlin** [comment]
Just installed DotKernel API v5 from scratch in a test directory - I can confirm that there are no issues with the database seeder.
Below is a step-by-step process of a fresh install starting from git cloning the project up to where it is ready to run.
(Click an item to reveal the command's output)

<details>
<summary>git clone https://github.com/dotkernel/api.git _api</summary>

> Cloning into '_api'...
> remote: Enumerating objects: 4838, done.
> remote: Counting objects: 100% (1091/1091), done.
> remote: Compressing objects: 100% (503/503), done.
> remote: Total 4838 (delta 558), reused 875 (delta 525), pack-reused 3747
> Receiving objects: 100% (4838/4838), 3.25 MiB | 3.70 MiB/s, done.
> Resolving deltas: 100% (2860/2860), done.

</details>

<details>
<summary>git status</summary>

> On branch 5.0
> Your branch is up to date with 'origin/5.0'.
> 
> nothing to commit, working tree clean

</details>

<details>
<summary>composer install</summary>

> No composer.lock file present. Updating dependencies to latest instead of installing from lock file. See https://getcomposer.org/install for more information.
> Loading composer repositories with package information
> Updating dependencies
> Lock file operations: 162 installs, 0 updates, 0 removals
>   - Locking amphp/amp (v2.6.4)
>   - Locking amphp/byte-stream (v1.8.2)
>   - Locking brick/math (0.12.1)
>   - Locking brick/varexporter (0.4.0)
>   - Locking composer/pcre (3.1.4)
>   - Locking composer/semver (3.4.0)
>   - Locking composer/xdebug-handler (3.0.5)
>   - Locking dealerdirect/phpcodesniffer-composer-installer (v0.7.2)
>   - Locking defuse/php-encryption (v2.4.0)
>   - Locking dnoegel/php-xdg-base-dir (v0.1.1)
>   - Locking doctrine/collections (2.2.2)
>   - Locking doctrine/common (3.4.4)
>   - Locking doctrine/data-fixtures (1.7.0)
>   - Locking doctrine/dbal (4.0.4)
>   - Locking doctrine/deprecations (1.1.3)
>   - Locking doctrine/event-manager (2.0.1)
>   - Locking doctrine/inflector (2.0.10)
>   - Locking doctrine/instantiator (2.0.0)
>   - Locking doctrine/lexer (3.0.1)
>   - Locking doctrine/migrations (3.8.0)
>   - Locking doctrine/orm (3.2.1)
>   - Locking doctrine/persistence (3.3.3)
>   - Locking dotkernel/dot-cache (4.1.0)
>   - Locking dotkernel/dot-cli (3.6.0)
>   - Locking dotkernel/dot-data-fixtures (1.2.3)
>   - Locking dotkernel/dot-dependency-injection (1.0.0)
>   - Locking dotkernel/dot-errorhandler (3.4.1)
>   - Locking dotkernel/dot-event (3.4.2)
>   - Locking dotkernel/dot-log (3.5.3)
>   - Locking dotkernel/dot-mail (4.2.2)
>   - Locking dotkernel/dot-response-header (3.3.1)
>   - Locking felixfbecker/advanced-json-rpc (v3.2.1)
>   - Locking felixfbecker/language-server-protocol (v1.5.2)
>   - Locking fidry/cpu-core-counter (1.1.0)
>   - Locking fig/http-message-util (1.1.5)
>   - Locking laminas/laminas-cli (1.10.0)
>   - Locking laminas/laminas-code (4.14.0)
>   - Locking laminas/laminas-coding-standard (2.5.0)
>   - Locking laminas/laminas-component-installer (3.4.0)
>   - Locking laminas/laminas-config (3.9.0)
>   - Locking laminas/laminas-config-aggregator (1.15.0)
>   - Locking laminas/laminas-development-mode (3.12.0)
>   - Locking laminas/laminas-diactoros (3.3.1)
>   - Locking laminas/laminas-escaper (2.13.0)
>   - Locking laminas/laminas-eventmanager (3.13.1)
>   - Locking laminas/laminas-filter (2.36.0)
>   - Locking laminas/laminas-http (2.19.0)
>   - Locking laminas/laminas-httphandlerrunner (2.10.0)
>   - Locking laminas/laminas-hydrator (4.15.0)
>   - Locking laminas/laminas-inputfilter (2.30.1)
>   - Locking laminas/laminas-loader (2.10.0)
>   - Locking laminas/laminas-log (2.17.0)
>   - Locking laminas/laminas-mail (2.25.1)
>   - Locking laminas/laminas-mime (2.12.0)
>   - Locking laminas/laminas-paginator (2.18.1)
>   - Locking laminas/laminas-permissions-acl (2.16.0)
>   - Locking laminas/laminas-permissions-rbac (3.6.0)
>   - Locking laminas/laminas-servicemanager (3.22.1)
>   - Locking laminas/laminas-stdlib (3.19.0)
>   - Locking laminas/laminas-stratigility (3.11.0)
>   - Locking laminas/laminas-text (2.11.0)
>   - Locking laminas/laminas-uri (2.11.0)
>   - Locking laminas/laminas-validator (2.60.0)
>   - Locking lcobucci/clock (2.2.0)
>   - Locking lcobucci/jwt (4.0.4)
>   - Locking league/event (2.2.0)
>   - Locking league/oauth2-server (8.4.2)
>   - Locking league/uri (6.8.0)
>   - Locking league/uri-interfaces (2.3.0)
>   - Locking mezzio/mezzio (3.19.0)
>   - Locking mezzio/mezzio-authentication (1.9.0)
>   - Locking mezzio/mezzio-authentication-oauth2 (2.8.0)
>   - Locking mezzio/mezzio-authorization (1.9.0)
>   - Locking mezzio/mezzio-authorization-acl (1.10.0)
>   - Locking mezzio/mezzio-authorization-rbac (1.7.0)
>   - Locking mezzio/mezzio-cors (1.12.1)
>   - Locking mezzio/mezzio-fastroute (3.11.0)
>   - Locking mezzio/mezzio-hal (2.9.0)
>   - Locking mezzio/mezzio-helpers (5.16.0)
>   - Locking mezzio/mezzio-problem-details (1.14.0)
>   - Locking mezzio/mezzio-router (3.17.0)
>   - Locking mezzio/mezzio-template (2.10.0)
>   - Locking mezzio/mezzio-tooling (2.9.0)
>   - Locking mezzio/mezzio-twigrenderer (2.15.0)
>   - Locking myclabs/deep-copy (1.12.0)
>   - Locking netresearch/jsonmapper (v4.4.1)
>   - Locking nikic/fast-route (v1.3.0)
>   - Locking nikic/php-parser (v4.19.1)
>   - Locking paragonie/random_compat (v9.99.100)
>   - Locking phar-io/manifest (2.0.4)
>   - Locking phar-io/version (3.2.1)
>   - Locking phpdocumentor/reflection-common (2.2.0)
>   - Locking phpdocumentor/reflection-docblock (5.3.0)
>   - Locking phpdocumentor/type-resolver (1.6.2)
>   - Locking phpstan/phpdoc-parser (1.5.1)
>   - Locking phpunit/php-code-coverage (10.1.15)
>   - Locking phpunit/php-file-iterator (4.1.0)
>   - Locking phpunit/php-invoker (4.0.0)
>   - Locking phpunit/php-text-template (3.0.1)
>   - Locking phpunit/php-timer (6.0.0)
>   - Locking phpunit/phpunit (10.5.24)
>   - Locking psr/cache (3.0.0)
>   - Locking psr/clock (1.0.0)
>   - Locking psr/container (1.1.2)
>   - Locking psr/event-dispatcher (1.0.0)
>   - Locking psr/http-client (1.0.3)
>   - Locking psr/http-factory (1.1.0)
>   - Locking psr/http-message (1.1)
>   - Locking psr/http-server-handler (1.0.2)
>   - Locking psr/http-server-middleware (1.0.2)
>   - Locking psr/link (1.1.1)
>   - Locking psr/log (1.1.4)
>   - Locking ramsey/collection (2.0.0)
>   - Locking ramsey/uuid (4.7.6)
>   - Locking ramsey/uuid-doctrine (2.1.0)
>   - Locking roave/psr-container-doctrine (5.1.0)
>   - Locking roave/security-advisories (dev-latest fed98f5)
>   - Locking sebastian/cli-parser (2.0.1)
>   - Locking sebastian/code-unit (2.0.0)
>   - Locking sebastian/code-unit-reverse-lookup (3.0.0)
>   - Locking sebastian/comparator (5.0.1)
>   - Locking sebastian/complexity (3.2.0)
>   - Locking sebastian/diff (5.1.1)
>   - Locking sebastian/environment (6.1.0)
>   - Locking sebastian/exporter (5.1.2)
>   - Locking sebastian/global-state (6.0.2)
>   - Locking sebastian/lines-of-code (2.0.2)
>   - Locking sebastian/object-enumerator (5.0.0)
>   - Locking sebastian/object-reflector (3.0.0)
>   - Locking sebastian/recursion-context (5.0.0)
>   - Locking sebastian/type (4.0.0)
>   - Locking sebastian/version (4.0.1)
>   - Locking slevomat/coding-standard (7.2.1)
>   - Locking spatie/array-to-xml (3.3.0)
>   - Locking squizlabs/php_codesniffer (3.10.1)
>   - Locking stella-maris/clock (0.1.7)
>   - Locking symfony/cache (v7.1.2)
>   - Locking symfony/cache-contracts (v3.5.0)
>   - Locking symfony/console (v7.1.2)
>   - Locking symfony/deprecation-contracts (v3.5.0)
>   - Locking symfony/event-dispatcher (v7.1.1)
>   - Locking symfony/event-dispatcher-contracts (v3.5.0)
>   - Locking symfony/filesystem (v7.1.2)
>   - Locking symfony/polyfill-ctype (v1.30.0)
>   - Locking symfony/polyfill-intl-grapheme (v1.30.0)
>   - Locking symfony/polyfill-intl-idn (v1.30.0)
>   - Locking symfony/polyfill-intl-normalizer (v1.30.0)
>   - Locking symfony/polyfill-mbstring (v1.30.0)
>   - Locking symfony/polyfill-php72 (v1.30.0)
>   - Locking symfony/polyfill-php80 (v1.30.0)
>   - Locking symfony/process (v6.4.8)
>   - Locking symfony/service-contracts (v3.5.0)
>   - Locking symfony/stopwatch (v7.1.1)
>   - Locking symfony/string (v7.1.2)
>   - Locking symfony/var-exporter (v7.1.2)
>   - Locking theseer/tokenizer (1.2.3)
>   - Locking twig/twig (v3.10.3)
>   - Locking vimeo/psalm (5.25.0)
>   - Locking webimpress/coding-standard (1.3.2)
>   - Locking webimpress/safe-writer (2.2.0)
>   - Locking webmozart/assert (1.11.0)
>   - Locking willdurand/negotiation (3.1.0)
> Writing lock file
> Installing dependencies from lock file (including require-dev)
> Package operations: 162 installs, 0 updates, 0 removals
>   - Installing laminas/laminas-component-installer (3.4.0): Extracting archive
>   - Installing squizlabs/php_codesniffer (3.10.1): Extracting archive
>   - Installing dealerdirect/phpcodesniffer-composer-installer (v0.7.2): Extracting archive
>   - Installing brick/math (0.12.1): Extracting archive
>   - Installing composer/pcre (3.1.4): Extracting archive
>   - Installing paragonie/random_compat (v9.99.100): Extracting archive
>   - Installing defuse/php-encryption (v2.4.0): Extracting archive
>   - Installing doctrine/deprecations (1.1.3): Extracting archive
>   - Installing doctrine/collections (2.2.2): Extracting archive
>   - Installing doctrine/inflector (2.0.10): Extracting archive
>   - Installing doctrine/instantiator (2.0.0): Extracting archive
>   - Installing doctrine/lexer (3.0.1): Extracting archive
>   - Installing symfony/var-exporter (v7.1.2): Extracting archive
>   - Installing symfony/deprecation-contracts (v3.5.0): Extracting archive
>   - Installing psr/container (1.1.2): Extracting archive
>   - Installing symfony/service-contracts (v3.5.0): Extracting archive
>   - Installing psr/cache (3.0.0): Extracting archive
>   - Installing symfony/cache-contracts (v3.5.0): Extracting archive
>   - Installing psr/log (1.1.4): Extracting archive
>   - Installing symfony/cache (v7.1.2): Extracting archive
>   - Installing dotkernel/dot-cache (4.1.0): Extracting archive
>   - Installing laminas/laminas-stdlib (3.19.0): Extracting archive
>   - Installing laminas/laminas-servicemanager (3.22.1): Extracting archive
>   - Installing symfony/polyfill-ctype (v1.30.0): Extracting archive
>   - Installing webmozart/assert (1.11.0): Extracting archive
>   - Installing symfony/polyfill-php80 (v1.30.0): Extracting archive
>   - Installing psr/event-dispatcher (1.0.0): Extracting archive
>   - Installing symfony/event-dispatcher-contracts (v3.5.0): Extracting archive
>   - Installing symfony/event-dispatcher (v7.1.1): Extracting archive
>   - Installing symfony/polyfill-mbstring (v1.30.0): Extracting archive
>   - Installing symfony/polyfill-intl-normalizer (v1.30.0): Extracting archive
>   - Installing symfony/polyfill-intl-grapheme (v1.30.0): Extracting archive
>   - Installing symfony/string (v7.1.2): Extracting archive
>   - Installing symfony/console (v7.1.2): Extracting archive
>   - Installing laminas/laminas-cli (1.10.0): Extracting archive
>   - Installing dotkernel/dot-cli (3.6.0): Extracting archive
>   - Installing doctrine/event-manager (2.0.1): Extracting archive
>   - Installing doctrine/persistence (3.3.3): Extracting archive
>   - Installing doctrine/dbal (4.0.4): Extracting archive
>   - Installing doctrine/orm (3.2.1): Extracting archive
>   - Installing doctrine/data-fixtures (1.7.0): Extracting archive
>   - Installing dotkernel/dot-data-fixtures (1.2.3): Extracting archive
>   - Installing dotkernel/dot-dependency-injection (1.0.0): Extracting archive
>   - Installing psr/http-message (1.1): Extracting archive
>   - Installing psr/http-server-handler (1.0.2): Extracting archive
>   - Installing psr/http-server-middleware (1.0.2): Extracting archive
>   - Installing psr/http-factory (1.1.0): Extracting archive
>   - Installing mezzio/mezzio-template (2.10.0): Extracting archive
>   - Installing fig/http-message-util (1.1.5): Extracting archive
>   - Installing mezzio/mezzio-router (3.17.0): Extracting archive
>   - Installing laminas/laminas-escaper (2.13.0): Extracting archive
>   - Installing laminas/laminas-stratigility (3.11.0): Extracting archive
>   - Installing laminas/laminas-diactoros (3.3.1): Extracting archive
>   - Installing laminas/laminas-httphandlerrunner (2.10.0): Extracting archive
>   - Installing mezzio/mezzio (3.19.0): Extracting archive
>   - Installing laminas/laminas-log (2.17.0): Extracting archive
>   - Installing symfony/polyfill-php72 (v1.30.0): Extracting archive
>   - Installing symfony/polyfill-intl-idn (v1.30.0): Extracting archive
>   - Installing laminas/laminas-validator (2.60.0): Extracting archive
>   - Installing laminas/laminas-mime (2.12.0): Extracting archive
>   - Installing laminas/laminas-loader (2.10.0): Extracting archive
>   - Installing laminas/laminas-mail (2.25.1): Extracting archive
>   - Installing laminas/laminas-eventmanager (3.13.1): Extracting archive
>   - Installing dotkernel/dot-event (3.4.2): Extracting archive
>   - Installing dotkernel/dot-mail (4.2.2): Extracting archive
>   - Installing dotkernel/dot-log (3.5.3): Extracting archive
>   - Installing dotkernel/dot-errorhandler (3.4.1): Extracting archive
>   - Installing psr/http-client (1.0.3): Extracting archive
>   - Installing dotkernel/dot-response-header (3.3.1): Extracting archive
>   - Installing webimpress/coding-standard (1.3.2): Extracting archive
>   - Installing phpstan/phpdoc-parser (1.5.1): Extracting archive
>   - Installing slevomat/coding-standard (7.2.1): Extracting archive
>   - Installing laminas/laminas-coding-standard (2.5.0): Extracting archive
>   - Installing laminas/laminas-config (3.9.0): Extracting archive
>   - Installing webimpress/safe-writer (2.2.0): Extracting archive
>   - Installing nikic/php-parser (v4.19.1): Extracting archive
>   - Installing brick/varexporter (0.4.0): Extracting archive
>   - Installing laminas/laminas-config-aggregator (1.15.0): Extracting archive
>   - Installing laminas/laminas-development-mode (3.12.0): Extracting archive
>   - Installing laminas/laminas-uri (2.11.0): Extracting archive
>   - Installing laminas/laminas-http (2.19.0): Extracting archive
>   - Installing laminas/laminas-hydrator (4.15.0): Extracting archive
>   - Installing laminas/laminas-filter (2.36.0): Extracting archive
>   - Installing laminas/laminas-inputfilter (2.30.1): Extracting archive
>   - Installing laminas/laminas-paginator (2.18.1): Extracting archive
>   - Installing laminas/laminas-text (2.11.0): Extracting archive
>   - Installing psr/clock (1.0.0): Extracting archive
>   - Installing stella-maris/clock (0.1.7): Extracting archive
>   - Installing lcobucci/clock (2.2.0): Extracting archive
>   - Installing lcobucci/jwt (4.0.4): Extracting archive
>   - Installing league/event (2.2.0): Extracting archive
>   - Installing league/uri-interfaces (2.3.0): Extracting archive
>   - Installing league/uri (6.8.0): Extracting archive
>   - Installing mezzio/mezzio-authentication (1.9.0): Extracting archive
>   - Installing league/oauth2-server (8.4.2): Extracting archive
>   - Installing mezzio/mezzio-authentication-oauth2 (2.8.0): Extracting archive
>   - Installing mezzio/mezzio-authorization (1.9.0): Extracting archive
>   - Installing laminas/laminas-permissions-acl (2.16.0): Extracting archive
>   - Installing mezzio/mezzio-authorization-acl (1.10.0): Extracting archive
>   - Installing laminas/laminas-permissions-rbac (3.6.0): Extracting archive
>   - Installing mezzio/mezzio-authorization-rbac (1.7.0): Extracting archive
>   - Installing mezzio/mezzio-cors (1.12.1): Extracting archive
>   - Installing nikic/fast-route (v1.3.0): Extracting archive
>   - Installing mezzio/mezzio-fastroute (3.11.0): Extracting archive
>   - Installing willdurand/negotiation (3.1.0): Extracting archive
>   - Installing psr/link (1.1.1): Extracting archive
>   - Installing mezzio/mezzio-hal (2.9.0): Extracting archive
>   - Installing spatie/array-to-xml (3.3.0): Extracting archive
>   - Installing mezzio/mezzio-problem-details (1.14.0): Extracting archive
>   - Installing symfony/process (v6.4.8): Extracting archive
>   - Installing laminas/laminas-code (4.14.0): Extracting archive
>   - Installing mezzio/mezzio-tooling (2.9.0): Extracting archive
>   - Installing twig/twig (v3.10.3): Extracting archive
>   - Installing mezzio/mezzio-helpers (5.16.0): Extracting archive
>   - Installing mezzio/mezzio-twigrenderer (2.15.0): Extracting archive
>   - Installing phpdocumentor/reflection-common (2.2.0): Extracting archive
>   - Installing phpdocumentor/type-resolver (1.6.2): Extracting archive
>   - Installing phpdocumentor/reflection-docblock (5.3.0): Extracting archive
>   - Installing sebastian/version (4.0.1): Extracting archive
>   - Installing sebastian/type (4.0.0): Extracting archive
>   - Installing sebastian/recursion-context (5.0.0): Extracting archive
>   - Installing sebastian/object-reflector (3.0.0): Extracting archive
>   - Installing sebastian/object-enumerator (5.0.0): Extracting archive
>   - Installing sebastian/global-state (6.0.2): Extracting archive
>   - Installing sebastian/exporter (5.1.2): Extracting archive
>   - Installing sebastian/environment (6.1.0): Extracting archive
>   - Installing sebastian/diff (5.1.1): Extracting archive
>   - Installing sebastian/comparator (5.0.1): Extracting archive
>   - Installing sebastian/code-unit (2.0.0): Extracting archive
>   - Installing sebastian/cli-parser (2.0.1): Extracting archive
>   - Installing phpunit/php-timer (6.0.0): Extracting archive
>   - Installing phpunit/php-text-template (3.0.1): Extracting archive
>   - Installing phpunit/php-invoker (4.0.0): Extracting archive
>   - Installing phpunit/php-file-iterator (4.1.0): Extracting archive
>   - Installing theseer/tokenizer (1.2.3): Extracting archive
>   - Installing sebastian/lines-of-code (2.0.2): Extracting archive
>   - Installing sebastian/complexity (3.2.0): Extracting archive
>   - Installing sebastian/code-unit-reverse-lookup (3.0.0): Extracting archive
>   - Installing phpunit/php-code-coverage (10.1.15): Extracting archive
>   - Installing phar-io/version (3.2.1): Extracting archive
>   - Installing phar-io/manifest (2.0.4): Extracting archive
>   - Installing myclabs/deep-copy (1.12.0): Extracting archive
>   - Installing phpunit/phpunit (10.5.24): Extracting archive
>   - Installing ramsey/collection (2.0.0): Extracting archive
>   - Installing ramsey/uuid (4.7.6): Extracting archive
>   - Installing ramsey/uuid-doctrine (2.1.0): Extracting archive
>   - Installing symfony/stopwatch (v7.1.1): Extracting archive
>   - Installing doctrine/migrations (3.8.0): Extracting archive
>   - Installing doctrine/common (3.4.4): Extracting archive
>   - Installing roave/psr-container-doctrine (5.1.0): Extracting archive
>   - Installing roave/security-advisories (dev-latest fed98f5)
>   - Installing symfony/filesystem (v7.1.2): Extracting archive
>   - Installing netresearch/jsonmapper (v4.4.1): Extracting archive
>   - Installing fidry/cpu-core-counter (1.1.0): Extracting archive
>   - Installing felixfbecker/language-server-protocol (v1.5.2): Extracting archive
>   - Installing felixfbecker/advanced-json-rpc (v3.2.1): Extracting archive
>   - Installing dnoegel/php-xdg-base-dir (v0.1.1): Extracting archive
>   - Installing composer/xdebug-handler (3.0.5): Extracting archive
>   - Installing composer/semver (3.4.0): Extracting archive
>   - Installing amphp/amp (v2.6.4): Extracting archive
>   - Installing amphp/byte-stream (v1.8.2): Extracting archive
>   - Installing vimeo/psalm (5.25.0): Extracting archive
> 45 package suggestions were added by new dependencies, use `composer suggest` to see details.
> Generating autoload files
> 119 packages you are using are looking for funding.
> Use the `composer fund` command to find out more!
> PHP CodeSniffer Config installed_paths set to ../../laminas/laminas-coding-standard/src,../../slevomat/coding-standard,../../webimpress,../../webimpress/coding-standard/src

</details>

<details>
<summary>cp config/autoload/local.php.dist config/autoload/local.php</summary>
</details>

<details>
<summary>cp config/autoload/mail.local.php.dist config/autoload/mail.local.php</summary>
</details>

<details>
<summary>cp config/autoload/cors.local.php.dist config/autoload/cors.local.php</summary>
</details>

<details>
<summary>vim config/autoload/local.php</summary>
</details>

<details>
<summary>php vendor/bin/doctrine-migrations migrate</summary>

>  WARNING! You are about to execute a migration in database "dotkernel_api" that could result in schema changes and data loss. Are you sure you wish to continue? (yes/no) [yes]:
>  >
> 
> 
>  [OK] Already at the latest version ("Api\Migrations\Version20240613153602")

</details>

<details>
<summary>php bin/doctrine fixtures:execute</summary>

> Executing Api\Fixtures\AdminRoleLoader
> Executing Api\Fixtures\OAuthClientLoader
> Executing Api\Fixtures\OAuthScopeLoader
> Executing Api\Fixtures\UserRoleLoader
> Executing Api\Fixtures\AdminLoader
> Executing Api\Fixtures\UserLoader
> Fixtures have been loaded.
>                 .''
>       ._.-.___.' (`\
>      //(        ( `'
>     '/ )\ ).__. )
>     ' <' `\ ._/'\
>        `   \     \

</details>

<details>
<summary>php -S 0.0.0.0:8080 -t public</summary>

Request:

> [Wed Jul  3 06:28:05 2024] PHP 8.3.8 Development Server (http://0.0.0.0:8080) started
> [Wed Jul  3 06:28:08 2024] 127.0.0.1:40728 Accepted
> [Wed Jul  3 06:28:08 2024] 127.0.0.1:40728 [200]: GET /

Response:

> {"message":"DotKernel API version 5"}

</details>

**Comment by arhimede** [comment]
OK] Already at the latest version ("Api\Migrations\Version20240613153602")

Try to run doctrine over an empty new database please

**Comment by alexmerlin** [comment]
The above steps are in the order I executed them in.
I ran the migration script on an empty database (I previously dropped all tables from the database).
Next, I ran the database seeder script, which completed successfully.

Based on the error you are getting, I see 2 possible reasons:

- doctrine cache is outdated
- your copy of DotKernel API is not the latest version (make sure you have the [src/App/src/Entity/TimestampsTrait.php](https://github.com/dotkernel/api/blob/5.0/src/App/src/Entity/TimestampsTrait.php) file)
